android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -287,6 +289,7 @@ private void startScanningDisplay(long scanStartTime, boolean hasResults) {
     long elapsedMillis = new Date().getTime() - scanStartTime;
     if (elapsedMillis < FIRST_SCAN_TIME_MILLIS
         || (elapsedMillis < SECOND_SCAN_TIME_MILLIS && !hasResults)) {
+      mNearbyDeviceAdapter.clear();
```

Comment:
I think this only works because clearing the ListAdapter triggers the empty view in the containing ListFragment.  It's suspicious to me that we ever explicitly change the visibility of the ListView since this is supposed to be handled automatically, see https://developer.android.com/reference/android/app/ListFragment.html

If it's not possible to rely on this behavior then it's fine to do it manually, but we should change the id of the empty view in fragment_nearby_beacons.xml so it's not "@id/android:empty" since according to the docs that's what's causing the unexpected visibility changes.